NDB.Covid19/NDB.Covid19/Locales/ur.json

```diff
@@ -88,24 +88,32 @@
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_4_TITLE": "ایپ کے پیچھے ٹیکنالوجی",
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_4_CONTENT": "ایپ بلیو توتھ استعمال کرتی ہے اور ایپل اور گوگل کی طرف سے فون میں مہیا ٹیکنالوجی پر مبنی ہے۔ یہ ٹیکنالوجی انفیکشن سے واسطے کی اطلاع دینے والا نظام ہے۔ آپ اپنے موبائل کی سیٹنگز میں اسے آن یا آف کر سکتے ہیں۔",
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_4_CONTENT2": "آپکا فون ہر 15 منٹ بعد ایک اتفاقی بلیو ٹوتھ ID بناتا ہے۔ جب دو فون ایک دوسرے کے قریب ہوں تو وہ IDs کا تبادلہ کرتے ہیں۔",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_4_CONTENT3": "یہاں Smittestopp ایپ کے بارے میں مزید تفصیل پڑھیں  <a href=\"www.helsenorge.no/smittestopp\">www.helsenorge.no/smittestopp</a>\"",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_4_CONTENT3": "یہاں Smittestopp ایپ کے بارے میں مزید تفصیل پڑھیں  <a href=\"https://www.helsenorge.no/smittestopp\">www.helsenorge.no/smittestopp</a>",
```

Comment:
Navigate to the english version on helsenorge.
^change in both excel sheet and this .json